index.js

```diff
@@ -68,7 +69,7 @@ module.exports = {
   },
 
   validateDependencies() {
-    if ('bootstrap' in this.app.project.bowerDependencies()) {
+    if (EmberApp.env() !== 'test' && 'bootstrap' in this.app.project.bowerDependencies()) {
```

Comment:
It would throw the error (well, with the latest commit, it's just a warning) when running inside of our addon itself. Because it still has bootstrap in bower.json. But after think about this again, we actually can remove it from our own bower.json, right? Then I can drop this of course...